### PR TITLE
[docs] Add example for cookies settings in Chrome

### LINKED ISSUE #2944: Third party cookies are disabled for non-incognito when using selenium grid with standalone seleniarm/chromium container
**Describe the bug**
When running tests using selenium grid with standalone seleniarm/chromium docker container browser inside of the container has setting to forbid all third party cookies

For both chrome and chromium installed to desktop default configuration is to forbid third-party cookies only in incognito, but the container browser has different defaults for some reason 

This interferes with some tests which in non-prod environment require third party cookies for authentication

--headless mode allows to work around the issue but is not convenient when debugging is required

**To Reproduce**
1) Run tests using selenium grid and following docker container seleniarm/standalone-chromium:latest
2) Use some wait in tests, while tests are waiting - connect to selenium grid session and check browser cookie settings

**Actual behavior**
Browser is set to not allow any third-party cookies 

**Expected behavior**
Third-party cookies should be allowed at least for non-incognito windows

**Artifacts**
image
image
**Execution environment (please complete the following information):**
 - Vividus version: 0.4.6 & 0.4.12
 - OS: macOS 12.4 ARM
 - Java: 16.0.2+7-67

**Desktop Web (if applicable, please complete the following information):**
 - OS: latest seleniarm/chromium docker container
 - Browser: Chromium 
 - Version v.103.0.5060.53

**Additional context**
Solution using web driver configuration described here: https://stackoverflow.com/questions/48667852/how-to-change-allow-sites-to-save-and-read-cookie-data-recommended-and-or-bloc/67260066#67260066

Running docker container using following string: docker run -d -e SCREEN_WIDTH=1920 -e SCREEN_HEIGHT=1080 -p 4444:4444 --shm-size=2g seleniarm/standalone-chromium:latest

### PR DESCRIPTION
**Pull request by VolhaHurynovich**
Closes #2944

// 0 = allow third party 
// 1 = block third party
// 2 = block third party in incognito
```
Requested capabilities:
{
  "browserName" : "chrome",
  "browserVersion" : "103.0",
  "goog:chromeOptions" : {
    "args" : [ ],
    "extensions" : [ ],
    "prefs" : {
      "profile" : {
        "cookie_controls_mode" : 2
      }
    }
  },
  "platformName" : "LINUX"
}
```

### CODE REVIEW

File: vividus-plugin-web-app/src/main/java/org/vividus/selenium/WebDriverFactory.java [hunk 1]
```diff
@@ -85,6 +86,12 @@ protected DesiredCapabilities updateDesiredCapabilities(DesiredCapabilities desi
                     {
                         WebDriverConfiguration configuration = getWebDriverConfiguration(type, false);
                         ChromeOptions chromeOptions = new ChromeOptions();
+                        Map<String, Object> prefs = new HashMap<>();
+                        Map<String, Object> profile = new HashMap<>();
+                        profile.put("cookie_controls_mode", 2);
+                        prefs.put("profile", profile);
+                        chromeOptions.setExperimentalOption("prefs", prefs);
```
Comment (uarlouski) [inline vividus-plugin-web-app/src/main/java/org/vividus/selenium/WebDriverFactory.java:93]: 1) it should be configurable
2) we already have ability to set experimental options as per line 96:
```
web.driver.CHROME.experimental-options={...}
```

Comment (VolhaHurynovich) [reply vividus-plugin-web-app/src/main/java/org/vividus/selenium/WebDriverFactory.java]: thank you, done

**Comment by ikalinin1** [comment]
Why do we need to put it in our profile?

File: vividus-plugin-web-app/src/main/resources/properties/profile/web/docker/profile.properties [hunk 2]
```diff
@@ -2,3 +2,5 @@ selenium.grid.enabled=true
 selenium.grid.capabilities.platformName=linux
 selenium.grid.capabilities.browserName=${selenium.browser}
 selenium.grid.url=http://localhost:4444/wd/hub
+
+web.driver.CHROME.experimental-options={"prefs": {"profile": {"cookie_controls_mode": 2}}}
```
Comment (valfirst) [inline vividus-plugin-web-app/src/main/resources/properties/profile/web/docker/profile.properties:6]: it should be moved to documentation, create a new section, e.g. `How to configure Chrome browser settings`

Comment (VolhaHurynovich) [reply vividus-plugin-web-app/src/main/resources/properties/profile/web/docker/profile.properties]: done

**Review by valfirst — CHANGES_REQUESTED** [review]

File: docs/modules/plugins/pages/plugin-web-app.adoc [hunk 3]
```diff
@@ -201,6 +201,19 @@ a|The command line (CLI) arguments to use when starting browser.
...
+
+.The settings to manage third party cookies
+[source,properties]
+----
+web.driver.CHROME.experimental-options={"prefs": {"profile": {"cookie_controls_mode": `$value`}}}
+----
+* `$value` - possible preference settings for third party cookies. One of:
+** `0` - Allow all cookies
+** `1` - Block third-party cookies
+** `2` - Block third-party cookies in Incognito
```
[the remark this answers is not in the document]
Comment (VolhaHurynovich) [reply docs/modules/plugins/pages/plugin-web-app.adoc]: done

[the remark this answers is not in the document]
Comment (VolhaHurynovich) [reply docs/modules/plugins/pages/plugin-web-app.adoc]: image

**Pull request merged**